docs/reference/management_commands.md

```diff
@@ -52,12 +52,31 @@ This command deletes old revisions which are not in moderation, live, approved t
 revision. If the `days` argument is supplied, only revisions older than the specified number of
 days will be deleted.
 
-If the `pages` argument is supplied, only revisions of page models will be deleted. If the `non-pages` argument is supplied, only revisions of non-page models will be deleted. If both or neither arguments are supplied, revisions of all models will be deleted.
+If the `pages` argument is supplied, only revisions of page models will be deleted. If the `non-pages` argument is supplied, only revisions of non-page models will be deleted. If both or neither arguments are supplied, revisions of all models will be deleted. 
+If deletion of a revision is not desirable, mark `Revision` with `on_delete=models.PROTECT`. 
 
 ```{versionadded} 5.1
 Support for deleting revisions of non-page models is added.
 ```
 
+Note: Purging Stale Revisions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Stale revisions are revisions that are no longer associated with any live objects. These revisions can accumulate over time as content is updated and new revisions are created.`purge_revisions` management command is used to remove stale revisions from the database.
+    Running the `purge_revisions` command  will permanently delete stale revisions that are no longer referenced by any live content. One method to prevent purging stale revisions is using a ForeignKey with `on_delete=models.PROTECT` which will prevent all stale revisions being referenced by the foreign key from being deleted.
+    Using ForeignKey with `on_delete=models.PROTECT`
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Here's an example of using `on_delete=models.PROTECT` with a `ForeignKey` field in your model definition:
+    .. code-block:: python
+        from django.db import models
+        from wagtail.core.models import Revision
+        class MyModel(models.Model):
+            revision = models.ForeignKey(
+                Revision,
+                on_delete=models.PROTECT,
+                )
+            # Add other fields for your model
+    With this setup, when a `Revision` instance is referenced by the `revision` field of `MyModel`, the `purge_revisions` command will not delete that particular revision, ensuring its preservation even if it becomes stale.
+
```

Comment:
This documentation is written not within an `eval-rst` block, so using the RST syntax is not going to work (it has to be Markdown+MyST). You can see that this new section breaks the rest of the documentation for the management commands: https://wagtail--10961.org.readthedocs.build/en/10961/reference/management_commands.html#purge-revisions

Please build the docs locally by running `make html` or `make livehtml` in the `docs` directory before committing your changes. Make sure that the changes are rendered correctly.

For the content itself, we don't need to explain what the command does in here, as it has already been explained in the paragraphs above. Also, I think `ForeignKey` and `on_delete` are pretty standard in Django, so we don't really have to add an example (or at least not with a very generic `MyModel` like this). Let's opt for a simpler paragraph after the first paragraph.

```suggestion
```

Reply:
Okay, I'll build the changes and check. I'll figure out how to do it and make the fix.